diff --git a/src/key_value_store/kv_flat_btree_async.cc b/src/key_value_store/kv_flat_btree_async.cc
index f3b8051..f28dee1 100644
--- a/src/key_value_store/kv_flat_btree_async.cc
+++ b/src/key_value_store/kv_flat_btree_async.cc
@@ -15,8 +15,8 @@
 #include "key_value_store/kv_flat_btree_async.h"
 #include "key_value_store/kvs_arg_types.h"
 #include "include/rados/librados.hpp"
-#include "/usr/include/asm-generic/errno.h"
-#include "/usr/include/asm-generic/errno-base.h"
+#include "asm-generic/errno.h"
+#include "asm-generic/errno-base.h"
 #include "common/ceph_context.h"
 #include "common/Clock.h"
 #include "include/types.h"
